src/db_impl_gc.cc

```diff
@@ -102,6 +103,15 @@ Status TitanDBImpl::BackgroundGC(LogBuffer* log_buffer) {
       s = blob_gc_job.Finish();
     }
     blob_gc->ReleaseGcFiles();
+
+    if (blob_gc->trigger_next() &&
+        (bg_gc_scheduled_ - 1 + gc_queue_.size() <
```

Comment:
Should consider CF? If the queue is occupied with other CFs and there is no task of current CF, we should still add to GC queue.

Reply:
For my opinion, if we consider the CF, we have to add a variable in DBImpl or ColumnFamily to record bg_gc_scheduled_ of each CF, it makes code complicate. If a gc task is done by previous task, even it triggered nothing would happen, I think it's unnecessary to consider queue size, seems that add to queue directly is OK.